internal/util/state/state.go

```diff
@@ -32,7 +32,7 @@ type State struct {
 	// never persisted
 	TelemetryLocked   bool      `json:"-"`
 	Start             time.Time `json:"-"`
-	LatestVersion     string    `json:"-"` // empty if IsUpdateAvailable is false
+	LatestVersion     string    `json:"-"` // empty if telemetry is disabled; latest version reported by telemetry otherwise
 	HandlerVersion    string    `json:"-"` // may be empty
 	IsUpdateAvailable bool      `json:"-"`
```

Comment:
```suggestion
	UpdateAvailable bool      `json:"-"`
```

The old name was used to avoid a clash with a function that this PR removes

Reply:
fixed